### DISCUSSION TITLE
Tor not working

### DISCUSSION THREAD

**Discussion by Lucius-fade** [termux/termux-app#3378] [General]
Please someone help me to solve this issue?..![Screenshot_2023-05-06-10-21-30-553_com.termux.jpg](https://user-images.githubusercontent.com/132731072/236600661-3790a977-12a6-4921-916e-3b6e69527e74.jpg)

**Comment by sylirre** [comment]
Read your logs once more and you will see that Tor tells you that port 9050 is in use. You can't run two or more programs on same port.

And yes, `tor` is **working**. Another good reason to investigate issue on your device side.

If you use Orbot or Tor Browser, both must be shut down before using Tor inside Termux. Otherwise edit the torrc file located at `$PREFIX/etc/tor`. If you already running `tor` in another session, no action needed.

![Screenshot_20230507-104257](https://user-images.githubusercontent.com/107305601/236664701-71ecdd55-8a6a-4f34-b1cf-18ef18604140.png)

![Screenshot_20230507-104447](https://user-images.githubusercontent.com/107305601/236664718-b0d4b845-bafc-4705-8bed-894c59a59790.png)

**Reply by musab09077188053** [reply]
Hi
